Source/WebCore/crypto/algorithms/CryptoAlgorithmAESGCM.cpp

```diff
@@ -54,11 +53,7 @@ static inline bool usagesAreInvalidForCryptoAlgorithmAESGCM(CryptoKeyUsageBitmap
 static inline bool tagLengthIsValid(uint8_t tagLength)
 {
     using namespace CryptoAlgorithmAESGCMInternal;
-    for (size_t i = 0; i < sizeof(ValidTagLengths); i++) {
-        if (tagLength == ValidTagLengths[i])
-            return true;
-    }
-    return false;
+    return std::find(validTagLengths.begin(), validTagLengths.end(), tagLength) != validTagLengths.end();
```

Comment:
More modern way to write this is `std::ranges::contains(validTagLengths, tagLength)`, but not sure we can use that yet. It’s C++20, for what it’s worth.

Reply:
I'll try and see if the bots are happy. Thanks!

Reply:
Sadly, some of the bots didn't like it:
```
In file included from /Volumes/Data/worker/macOS-Ventura-Release-Build-EWS/build/WebKitBuild/Release/DerivedSources/WebCore/unified-sources/UnifiedSource118.cpp:2:
/Volumes/Data/worker/macOS-Ventura-Release-Build-EWS/build/Source/WebCore/crypto/algorithms/CryptoAlgorithmAESGCM.cpp:57:25: error: no member named 'contains' in namespace 'std::ranges'
```

Reply:
We can give up on it of course unless it's just a missing include.

Reply:
Actually, both `<ranges>` and `<algorithm>` were included so I don't think that was the issue. I'm trying with `std::ranges::find()` now as a fall back.

Reply:
`std::ranges::find()` looks good on the bots so far.